pkg/account/downloader/internal/http/groups.go

```diff
@@ -29,9 +28,6 @@ func (c *Client) GetGroups(ctx context.Context, accUUID string) ([]accountmanage
 	if err != nil {
 		return nil, err
 	}
-	if r != nil && int(r.Count) != len(r.Items) {
```

Comment:
As discussed directly, makes sense to me for the reason you mentioned in the PR description. 
Neither a user nor we can actually do anything about that error case. 

@jskelin is there something @Laubi and I are missing though? Is this a case you actually ran into while developing the download?